app/javascript/chat/compose.jsx

```diff
@@ -11,7 +11,6 @@ export default class Chat extends Component {
     handleKeyUp: PropTypes.func.isRequired,
     startEditing: PropTypes.bool.isRequired,
     markdownEdited: PropTypes.bool.isRequired,
-    editMessageHtml: PropTypes.string.isRequired,
```

Comment:
@benhalpern I got rid of this completely because the user will be able to see the message they're editing anyway in the textarea.

Can you tell me how I would make the textarea automatically focused if the user selects "Edit"?